tests/integration_tests/package_upload_rate_update.test.ts

```diff
@@ -238,4 +238,44 @@ describe('POST /package/byRegEx', () => {
   });
 });
 
+describe('POST /packages', () => {
+  test('Packages found 200', async() => {
+    const result = await request(app)
+      .post('/packages')
+      .send([{ Name: '*', Version: '1.0.0'}])
+      .set('X-Authorization', `bearer ${token}`)
+      .set('offset', '1')
+      .set('Content-Type', 'application/json');
+    expect(result.statusCode).toEqual(200);
+    expect(result.body).toEqual(
+      expect.arrayContaining([
+        expect.objectContaining({
+          Name: expect.any(String),
+          Version: expect.any(String),
+          ID: expect.any(String),
+        }),
+      ]),
+    );
+  });
+
+  test('Packages found query', async() => {
+    const result = await request(app)
+      .post('/packages')
+      .send([{ Name: 'cloudinary', Version: '^1.2.0'}])
```

Comment:
I don't think cloudinary is uploaded in these tests unless you seeded the database
Change this to underscore maybe, or nodejs-file-downloader

Reply:
maybe im wrong I've just been changing these tests a lot last few days, let me double check

Reply:
are you sure? I got the correct response when I went through the logs

Reply:
I did see it.

Reply:
I'll change it if it's an issue later.

Reply:
Yeah change it to underscore version "1.13.6". Cloudinary was meant to be a fail test, but it now passes and I'm going to be changing that - this is a bug on my end

Reply:
Nope not a me issue - I changed this yesterday morning. Rebase to main and update your test to underscore version 1.13.6.
When was the last time you rebased?

Reply:
> are you sure? I got the correct response when I went through the logs

Was correct but now is not because that test was changed. Cloudinary is not in the db upon running your tests anymore

Reply:
will update it

Reply:
I just noticed a small bug, I will update it for a secondary review